manifest-template.js

```diff
@@ -0,0 +1,85 @@
+#!/bin/node
+const fs = require('fs')
+
+const [hash, url] = process.argv.slice(2)
+
+const manifest = {
+    "app-id": "com.tutanota.Tutanota",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "19.08",
+    "branch": "stable",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "tutanota-desktop",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=host",
```

Comment:
Can you remove the `host` permission? Maybe to something like:

```diff
-          "--filesystem=host", 
+          "--filesystem=xdg-documents",
+          "--filesystem=xdg-pictures",
+          "--filesystem=xdg-videos",
```

I personally don't see the point of it having access to almost everything. It would be better to restrict it to directories that it really should have access.

Reply:
Which directories you "really access" when user picks file? (I know it *should* be handled by portal but I think Electron doesn't at the moment).

Reply:
Users will usually pick files from their Documents, Pictures, Videos, Pictures and Downloads folders. It's *very* rare to see a user pick a file directly from $HOME.

Anyway, allowing access to host when it shouldn't is unnecessary. At least reducing it to `--filesystem=home` or even to my proposal would be better options.

Reply:
I'm sure people will totally get it when they can only pick files from certain folders.
What about removable media? it's not included in $HOME I bet.

Generally we only need access to the user picks, config foldrer plus marker for the. running instance but as I picking is not working like it should yet I don't see a way to narrow it down. Would be glad to do so.

Reply:
For the removable media, isn't it `--device=all`?

I'll take a look later. The flatpak doesn't seem to be updating right now

Reply:
So... Isolation is a lie. 🤦🏼‍♀️

Reply:
@TheEvilSkeleton, @charlag 

2 Nov 2020:
> *Okey, **write** access to the full filesystem*

27 days later:
> [Tutanota provide a **backdoor**](https://www.heise.de/news/Gericht-zwingt-Mailprovider-Tutanota-zu-Ueberwachungsfunktion-4972460.html) [[eng](https://www.theregister.com/2020/12/08/tutanota_backdoor_court_order/)]

🤔👽🛸